src/ocean/math/IEEE.d

```diff
@@ -307,6 +307,7 @@ private:
            static assert(0, "Not yet supported");
        }
     }
+    deprecated
```

Comment:
Could someone with x86 assembly language knowledge take a look at `resetIeeeFlags()`, please ( @don-clugston-sociomantic pr @nemanja-boric-sociomantic )? It seems to me it would work with x86-64 but uses an orphan `version` identifier.

Reply:
It seems that that's still applicable even for x86-64 (as part of x87 FPU).

Reply:
That's right, there's no difference between x86 and x86-64 in this respect